driver/docker-container/driver.go

```diff
@@ -121,10 +122,18 @@ func (d *Driver) create(ctx context.Context, l progress.SubLogger) error {
 					Target: containerBuildKitRootDir,
 				},
 			},
+			Resources: container.Resources{
+				// Place all buildkit containers inside this cgroup so limits can be attached
+				// to all build activity on the host.
+				CgroupParent: "/docker/buildx",
+			},
 		}
 		if d.netMode != "" {
 			hc.NetworkMode = container.NetworkMode(d.netMode)
 		}
+		if d.cgroupParent != "" {
+			hc.CgroupParent = d.cgroupParent
```

Comment:
Probably, this should be set only when  `d.DockerAPI.Info.CgroupDriver == “cgroupfs”`